templates/taxi/index.html

```diff
@@ -10,4 +10,5 @@ <h2>Dynamic content</h2>
     <li><strong>Drivers:</strong> {{ num_drivers }}</li>
     <li><strong>Manufacturers:</strong> {{ num_manufacturers }}</li>
   </ul>
+  <p>You have visited this page {{ num_visits }} times</p>
```

Comment:
make it change between `time` and `times`

Reply:
fix that